tools/fiptool/tbbr_config.h

```diff
@@ -9,7 +9,7 @@
 
 #include <stdint.h>
 
-#include "uuid.h"
+#include <uuid.h>
```

Comment:
stdint.h and uuid.h belong to different categories (in the sense how public it is).
I think a blank line is OK.